network.go

```diff
@@ -1148,3 +1149,25 @@ func (n *network) IpamConfig() (string, []*IpamConf, []*IpamConf) {
 
 	return n.ipamType, v4L, v6L
 }
+
+func (n *network) IpamInfo() ([]*IpamInfo, []*IpamInfo) {
+	n.Lock()
```

Comment:
Please re-use `func (n *network) getIPInfo(ipVer int) []*IpamInfo`, so function would reduce to:
```
{
    return getIPInfo(4), getIPInfo(6)
}
```

I think it is ok, we don't need a deep copy of the fields.